docs/docs/docs/metrics.md

```diff
@@ -40,7 +39,7 @@ object MetricApp extends IOApp {
       metricPrefix = Some("my_app"),
 
       // the address your Datadog agent is listening on
-      agentHost = new InetSocketAddress("localhost", 8125),
+      agentHost = SocketAddress(ip"127.0.0.1", port"8080"),
```

Comment:
how come the port has changed?

Reply:
oh yes I should put that back I think :thinking: